packages/identity/src/identity.js

```diff
@@ -286,7 +296,11 @@ export async function getPublicIdentity(tankerIdentity: b64string): Promise<b64s
   }
 
   if (identity.public_signature_key && identity.public_encryption_key) {
-    const { trustchain_id, target, value, public_signature_key, public_encryption_key } = identity; // eslint-disable-line camelcase
+    const { trustchain_id, public_signature_key, public_encryption_key } = identity; // eslint-disable-line camelcase
+    let { target } = identity;
+    const value = await _getPublicHashedValueFromSecretProvisional(identity);
+    if (target === 'email') // For compatibility reasons, for hashed_email pub prov ids, the private is just 'email'
```

Comment:
Not sure I understand this comment. It's the same for phone, there's no compatibility issue here, is it?

Reply:
There used to be, the comment is stale. Fixed.